client/src/ui/modules/navigation/QuestMenu.tsx

```diff
@@ -102,96 +105,101 @@ export const QuestsMenu = ({ unclaimedQuestsCount }: { unclaimedQuestsCount: num
-        onMouseLeave={() => !worldLoading && setTooltip(null)}
-        disabled={currentQuest?.status !== QuestStatus.Completed || worldLoading}
-      >
-        {worldLoading ? "Loading..." : "Claim"}
-      </Button>
-
-      <div className="h-full flex items-center">
-        <div className="h-[80%] w-px bg-gold/30 mx-2" />
-      </div>
-
-      <Button
-        onClick={() => handleStart()}
-        variant="outline"
-        disabled={
-          (currentQuest?.status === QuestStatus.Completed && currentQuest.id !== QuestType.Settle) || worldLoading
-        }
-        className={clsx("tutorial-selector relative text-sm capitalize", {
-          "!border-gold/70 !text-brown !bg-gold hover:!bg-gold/70 animate-pulse hover:animate-none":
-            currentQuest?.status !== QuestStatus.Completed && !worldLoading,
-        })}
-      >
-        <span className="font-semibold">{worldLoading ? "Loading..." : currentQuest?.name}</span>
+    unclaimedQuestsCount > 0 &&
+    !isWorldLoading && (
+      <div className="absolute right-0 px-4 top-full mt-2">
         <div
-          className={clsx(
-            "absolute rounded-full border border-green/30 bg-green/90 text-brown px-1.5 md:px-2 text-[0.6rem] md:text-xxs z-[100] font-bold -top-1 -right-1",
-            {
-              "animate-bounce": !worldLoading,
-            },
-          )}
+          className={clsx("flex gap-2 bg-brown/90 border border-gold/30 rounded-full px-4 h-10 md:h-12 py-2", {
+            "opacity-50 pointer-events-none": worldLoading,
+          })}
         >
-          {unclaimedQuestsCount}
-        </div>
-      </Button>
-
-      <div className="h-full flex items-center">
-        <div className="h-[70%] w-px bg-gold/30 mx-2" />
-      </div>
-
-      {skipQuest ? (
-        <div className="flex flex-row gap-4">
-          <Button
-            className="text-sm font-semibold capitalize"
-            onClick={handleClaimAllQuests}
-            variant="red"
-            disabled={worldLoading}
-          >
-            {worldLoading ? "Loading..." : "Skip All Quests"}
-          </Button>
           <Button
-            className="text-sm font-semibold capitalize"
+            variant="outline"
+            isLoading={isLoading}
+            className={clsx("claim-selector text-sm !font-bold capitalize", {
+              "!border-gold/70 !text-brown !bg-gold hover:!bg-gold/70 animate-pulse hover:animate-none":
+                currentQuest?.status === QuestStatus.Completed && !worldLoading,
+            })}
             onClick={handleClaimQuest}
-            variant="red"
-            disabled={worldLoading}
+            onMouseEnter={handleClaimMouseEnter}
+            onMouseLeave={() => !worldLoading && setTooltip(null)}
+            disabled={currentQuest?.status !== QuestStatus.Completed || worldLoading}
           >
-            {worldLoading ? "Loading..." : "Confirm"}
+            {worldLoading ? "Loading..." : "Claim"}
```

Comment:
worldLoading has to be false now for the component to render, so we should remove these